web_console/customizing-the-web-console.adoc

```diff
@@ -26,26 +30,40 @@ If a resource has a message about resource limitations or quotas being reached,
 
 If using *List View* from the *View Shortcuts* menu, resources appear as a list. The *Alerts* column indicates if a message is available.
 
+// cannot create resource "secrets" in openshift-config
```

Comment:
Nit: I would prefer to see the repetitive comments consolidated and summarized for legibility.